test/ir/pir/CMakeLists.txt

```
@@ -39,3 +39,7 @@ py_test_modules(
   FLAGS_pir_subgraph_saving_dir=${CMAKE_CURRENT_SOURCE_DIR})
 
 add_subdirectory(fused_pass)
+
+if(WITH_DISTRIBUTE)
+  add_subdirectory(translator)
+endif()
```

translator的单测为什么要通过 WITH_DISTRIBUTE控制，这里应该只有分布式算子单测通过这个宏控制
```suggestion
add_subdirectory(translator)
```

---

这里主要是有些CI没有打开WITH_DISTRIBUTE，执行单侧会造成xx_op没有注册，已在`test/ir/pir/translator/CMakeLists.txt`添加判断